metadata-io/src/main/java/com/linkedin/metadata/entity/ebean/EbeanAspectDao.java

```diff
@@ -498,15 +498,19 @@ public Map<String, Long> getNextVersions(@Nonnull final String urn, @Nonnull fin
     if (exp == null) {
       return result;
     }
+    // Ensure Ordering of Aspect version
+    exp.orderBy("version");
     List<EbeanAspectV2.PrimaryKey> dbResults = exp.endOr().findIds();
 
     for (EbeanAspectV2.PrimaryKey key: dbResults) {
+      log.debug("Reading from DB - Urn: {}, Aspect: {}, currVersion: {}", urn, key.getAspect(), key.getVersion());
```

Comment:
Can we remove these logs? Only cause this operation is called pretty often and may flood the logs unnecessarily. I think given your orderBy change we hopefully should not need to revisit this :p

Reply:
Removed the debug logs